js/components/LeftNavArrow/styles.js

```diff
@@ -0,0 +1,17 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  container: {
+
+    position: 'absolute',
+    left: 10,
+    top: '50%'
+  },
+  arrowContainer: {
```

Comment:
Remove empty style rule.

Reply:
Removed empty rule from Styles and Left and Right Nav file.